test/unit/io_managers_test.py

```diff
@@ -283,8 +284,7 @@ def test_migrations_match_metadata(tmp_path, monkeypatch):
     # run all the migrations on a fresh DB at tmp_path/pudl.sqlite
     alembic.config.main(["upgrade", "head"])
 
-    pkg = Package.from_resource_ids()
-    PudlSQLiteIOManager(base_dir=tmp_path, db_name="pudl", package=pkg)
+    PudlSQLiteIOManager(base_dir=tmp_path, db_name="pudl", package=PUDL_PACKAGE)
```

Comment:
I ultimately didn't try adding the encoding step to the IO Manager, because I think there are probably some EIA-861 tables with names that match the encoding tables FKs, but which aren't ready to be encoded yet, and because I saw there was already an encoding step happening in the FERC transforms.